.ebextensions/04-cerbot.config

```diff
@@ -0,0 +1,10 @@
+packages:
+  yum:
+    # mod_ssl for apache
+    mod24_ssl.x86_64: []
+
+container_commands:
+  10_installCertbot:
+    command: "wget https://dl.eff.org/certbot-auto;chmod a+x certbot-auto"
+#  20_getCert:
+#    command: "sudo ./certbot-auto certonly --debug --non-interactive --email ${CERT_EMAIL}  --domains ${CERT_DOMAIN} --agree-tos"
```

Comment:
It it possible to, rather than disable this command, have it as an option somehow?

Reply:
I don't know how. I should delete the whole cerbot thing. We don't need it anymore with the AWS certificate manager.

Reply:
Yeah, let's just delete it for now. Can always find it again in the history if we need to re-insert.